doc/nrf/ug_bt_coex.rst

```diff
@@ -38,11 +38,26 @@ Make sure that the following KConfig options are enabled:
 Configuring 3-wire coexistence
 ******************************
 
-Configuration is set using the devicetree (DTS).
 For more information about devicetree overlays, see :ref:`zephyr:use-dt-overlays`.
 A sample devicetree overlay is available at :file:`samples/bluetooth/radio_coex_3wire/boards/nrf52840dk_nrf52840.overlay`.
 The elements are described in the bindings: :file:`dts/bindings/radio_coex/sdc-radio-coex-three-wire.yaml`.
 
+Priorities and scanner mode can be configured or updated at run time.
+
+Per-state priorities
```

Comment:
```suggestion
Role-based priorities
```
Also, you can make this and the other new heading subheadings since they fit under Configuring 3-wire coexistence.